Tests/Sprint_2/test_Sign_Up_Page_Test.py

```diff
@@ -0,0 +1,86 @@
+            driver.get("http://localhost:8000/")
+            driver.find_element(By.ID, "id_new_account_button").click()
+            driver.find_element(By.ID, "id_username").send_keys("jefferson")
+            driver.find_element(By.ID, "id_password1").send_keys("passwordg")
+            driver.find_element(By.ID, "id_password2").send_keys("passwordg")
+            driver.find_element(By.ID, "id_submit_new_account_button").click()
+            self.assertEquals(
+                "The dashboard is currently in mode All Events",
+                driver.find_element(By.ID, "Current Filter Mode").text
+            )
+
+    # TC 2: create new account with weak password and get denied
+    def test_weak_password(self):
+        with driver_manager() as driver:
+            driver.get("http://localhost:8000/")
+            driver.find_element(By.ID, "id_new_account_button").click()
+            driver.find_element(By.ID, "id_username").send_keys("jefferson")
+            driver.find_element(By.ID, "id_password1").send_keys("1")
+            driver.find_element(By.ID, "id_password2").send_keys("1")
+            driver.find_element(By.ID, "id_submit_new_account_button").click()
+            self.assertEquals(
+                "Sign up",
+                driver.find_element(By.ID, "id_submit_new_account_button").text
+            )
+
+    # TC 3: create new account with a already used username and get denied
+    def test_already_used_name(self):
+        with driver_manager() as driver:
+            driver.get("http://localhost:8000/")
+            driver.find_element(By.ID, "id_new_account_button").click()
+            driver.find_element(By.ID, "id_username").send_keys("admin")
+            driver.find_element(By.ID, "id_password1").send_keys("passwordg")
+            driver.find_element(By.ID, "id_password2").send_keys("passwordg")
+            driver.find_element(By.ID, "id_submit_new_account_button").click()
+            self.assertEquals(
+                "Sign up",
+                driver.find_element(By.ID, "id_submit_new_account_button").text
+            )
+
+    # TC 4: create new account with two different passwords and get denied
+    def test_two_passwords(self):
+        with driver_manager() as driver:
+            driver.get("http://localhost:8000/")
+            driver.find_element(By.ID, "id_new_account_button").click()
+            driver.find_element(By.ID, "id_username").send_keys("jefferson")
+            driver.find_element(By.ID, "id_password1").send_keys("passwordg")
+            driver.find_element(By.ID, "id_password2").send_keys("pasbananag")
+            driver.find_element(By.ID, "id_submit_new_account_button").click()
+            self.assertEquals(
+                "Sign up",
+                driver.find_element(By.ID, "id_submit_new_account_button").text
+            )
+
+# placeholder
+# TC 5: create new account with a long username and get denied, currently not working reportedly (but merge still please)

```

Comment:
Also I would argue this should be uncommented because this is a test that we do need to pass at some point and there's no point in leaving it commented out.

Reply:
Ok, I opened up that test case